package.json

```diff
@@ -16,7 +16,7 @@
     "emscripten": "3.1.54",
     "llvm": "17.0.6",
     "engines": {
-        "node": ">=14.18.2 <22"
+        "node": ">=16"
```

Comment:
no problem with this, just note that our build scripts don't work on node 22

Reply:
Any idea why? I run my local build on node 22.